cognite/client/data_classes/data_modeling/views.py

```diff
@@ -256,114 +255,90 @@ def __init__(
         self.include_global = include_global
 
 
-@dataclass
-class ViewDirectRelation(DirectRelation):
-    source: Optional[ViewId] = None
```

Comment:
Was this not in use?

Reply:
I use this in `pygen`, so that will be a breaking change for that package.

Reply:
We don't need ContainerDirectRelation or ViewDirectRelation. Those are just DirectRelation really. Added some type aliases for backwards compatiblity.

Reply:
Slight breaking change in that `source` is moved from `ViewDirectRelation` to `MappedProperty`, so that it's on the same level on input and output. This is just plaing wrong in the API, and I'd like it to make sense in the SDK.